python/pytorchserver/pytorchserver/model.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfserving
+import numpy as np
+import os
+from typing import List, Any
+import torch
+from torch.autograd import Variable
+import importlib
+import shutil
+import sys
+
+PYTORCH_FILE = "model.pt"
+
+class PyTorchModel(kfserving.KFModel):
+    def __init__(self, name: str, model_class_name: str, model_class_file: str, model_dir: str):
+        super().__init__(name)
+        self.name = name
+        self.model_class_name = model_class_name
+        self.model_class_file = model_class_file
+        self.model_dir = model_dir
+        self.ready = False
+
+    def load(self):
+        model_file = os.path.join(
```

Comment:
Can we copy the model dir and discover the file? Perhaps just pick the first one (or all n) that we find?

Reply:
Goes back to previous comment, but I would prefer having a cleaner interface where the responsibility of passing model class file and class name is upto the client - reduces error pssobilities

Reply:
Also there can be multiple .py files, and will be again very error-prone to know which one is the actual class file

Reply:
Done